docs/pages/access-controls/device-trust.mdx

```diff
@@ -1,31 +1,94 @@
 ---
 title: Device Trust (Preview)
-description: Use and enforce trusted devices with Teleport
+description: Teleport Device Trust Concepts
 layout: tocless-doc
+videoBanner: gBQyj_X1LVw
 ---
 
 <Admonition type="warning">
-Device Trust is currently in Preview mode.
+  Device Trust is currently in Preview mode and supports following components:
 
-Device Trust requires Teleport Enterprise v12+.
+  - User devices: macOS.
+  - Teleport client: `tsh` and Teleport connect.
+  - Resources: SSH, Database and Kubernetes.
+
+  Support for Windows device is available in version 13.0 and above.
+
+  Support for other operating systems, access from Web UI and application
+  access is planned for upcoming Teleport versions.
 </Admonition>
 
+## Concepts
+
 Device Trust allows Teleport admins to enforce the use of trusted devices.
 Resources protected by the device mode "required" will enforce the use of a
 trusted device, in addition to establishing the user's identity and enforcing
 the necessary roles. Furthermore, users using a trusted device leave audit
 trails that include the device's information.
 
-The device trust preview works only with macOS devices and supports the
-following Teleport features:
+Device Trust requires two of the following steps to have been configured:
+
+- Trusted device registered and enrolled with Teleport.
+- Device enforcement mode configured via either a role or a cluster-wide config.
```

Comment:
In v12, we only support cluster-wide config, right? I'm pretty sure role enforcement came in v13.

Reply:
v12 supports both cluster-wide and RBAC. v12 doc link for reference - https://goteleport.com/docs/ver/12.x/access-controls/device-trust/guide/#optional-role-based-trusted-device-enforcement